surveys/views.py

```diff
@@ -3,5 +3,5 @@
 
 
 def index(request):
-    question = Question.objects.all()[0]
+    question = Question.objects.order_by('?').first()
```

Comment:
This could be a source of performance issue according to django docs [here](https://docs.djangoproject.com/en/2.2/ref/models/querysets/#order-by) might want to allocate some time to investigate other solutions as we move further